code/datums/mind.dm

```diff
@@ -44,7 +44,12 @@
 	var/list/datum/objective/special_verbs = list()
 	var/list/targets = list()
 
-	var/has_been_rev = 0//Tracks if this mind has been a rev or not
+	///Tracks if this mind has been a rev or not
+	var/has_been_rev = FALSE
+	///Tracks if headrevs are on a conversion cooldown or not
+	var/conversion_on_cooldown = FALSE
+	///How long the cooldown timer is for conversion
+	var/conversion_cooldown_time = 20 SECONDS
```

Comment:
I really do not like how these are tacked onto the mind.

Reply:
Indeed, I'm not sure where else to put it, unfortunately. It can't go on the flash since people will just carry a bag of them, and there isn't a rev datum that is assigned to the mind.

Reply:
You could make it a status effect. That way people know when they are on cooldown aswell.

Reply:
You can make it a static thing on the flash item. Have an assoc list with the mind (UID or so) or so as key and the value being the cooldown. That way it stays on the flash item and not on the mind and it is global for all flashes

Reply:
Seconding @hal9000PR 's status effect suggestion. Let's not introduce more variables in the mind that are used in only one game mode that's not even on rotation.